jvcl/run/JvInterpreter_Math.pas

```diff
@@ -353,7 +353,9 @@ procedure RegisterJvInterpreterAdapter(JvInterpreterAdapter: TJvInterpreterAdapt
     AddConst(cMath, 'MaxSingle', MaxSingle);
     AddConst(cMath, 'MinComp', MinComp);
     AddConst(cMath, 'MinDouble', MinDouble);
+    {$WARN SYMBOL_PLATFORM OFF} // MinExtended is platform specific
     AddConst(cMath, 'MinExtended', MinExtended);
+    {$WARN SYMBOL_PLATFORM ON}
```

Comment:
You are not restoring the previous state here

Reply:
How do I restore the previous state?

Reply:
I would use

    {$WARN SYMBOL_PLATFORM DEFAULT}

But I'm not sure `DEFAULT` is supported by all compilers that support `SYMBOL_PLATFORM`

Reply:
I'm not sure about the goal of the last part regarding JvInterpreter_Math.pas. If I were I might be able to propose a solution. Shall plattform specific code warnings be suppressed with that?

Reply:
Instead of trying to restore the SYMBOL_PLAFORM warning, we can use the MinExtended80 if it is supported and the target platform supports it and fall back to MinDouble if it doesn't.